blitz_api/models.py

```diff
@@ -415,6 +452,51 @@ def credit_tickets(self, nb_tickets: int):
         self.tickets += nb_tickets
         self.save()
 
+    def generate_tomato_field_matrix_cell(self):
+        """
+        Generate the content of a cell for the tomato matrix
+        """
+
+        rdm = random.choices(
+            [c['tile'] for c in self.TOMATO_MATRIX_CELLS],
+            weights=(c['weight'] for c in self.TOMATO_MATRIX_CELLS)
+        )
+        return rdm[0]
+
+    def generate_tomato_field_matrix(self,
+                                     line_size=TOMATO_MATRIX_LINE_SIZE,
+                                     column_size=TOMATO_MATRIX_COLUMN_SIZE):
+        """
+        Generate a matrix with common cells and random cells, based on visual
+        objects
+        :params line_size: size of the matrix line, default 5
+        :params column_size: size of the matrix column, default 5
+        return the generated matrix as a list of list
+        """
+        matrix = []
+        for m in range(0, column_size):
+            matrix_line = []
+            for n in range(0, line_size):
+                cell_value = {}
+                cell_position = (n, m)
+                if cell_position in self.TOMATO_MATRIX_COMMON_CELLS:
+                    cell_value = self.TOMATO_MATRIX_COMMON_CELLS[cell_position]
+                elif cell_position in self.TOMATO_MATRIX_RANDOM_CELLS:
+                    cell_value = self.generate_tomato_field_matrix_cell()
```

Comment:
I spoke with Melanie to review the PR and `COMMON` and `RANDOM` cells's number doesn't match customer requirement.